File: FireSaverSPA/src/app/Models/UserService/userInfoDto.ts
import { Postion } from "../PointService/pointDtos";

export class UserInfoDto {
    id: number = -1;
    name: string = "";
    surname: string = "";
    patronymic: string = "";
    mail: string = "";
    telephoneNumber: string = "";
    dob: Date = new Date();
    lastSeenBuildingPosition: Postion = null;
}

File: FireSaverApi/Dtos/UserDtos/UserInfoDto.cs
using System;
using System.Collections.Generic;
using FireSaverApi.Dtos.BuildingDtos;

namespace FireSaverApi.Dtos
{
    public class UserInfoDto
    {
        public int Id { get; set; }
        public List<string> RolesList { get; set; }
        public string Name { get; set; }
        public string Surname { get; set; }
        public string Patronymic { get; set; }
        public string Mail { get; set; }
        public string TelephoneNumber { get; set; }
        public BuildingInfoDto ResponsibleForBuilding { get; set; }
        public DateTime DOB { get; set; }
        public PositionDto LastSeenBuildingPosition { get; set; }
    }
}